### PR TITLE
Wen - pizzeria

### PR DESCRIPTION
**Pull request by wwenzz**
## Netlify link
https://wens-javascript-pizzeria.netlify.app/

## Collaborators
[wwenzz]

### CODE REVIEW

File: pull_request_template.md
Comment (AntonellaMorittu) [inline pull_request_template.md:1]: Please delete all files that are not useful for the application, like `pull_request_template.md`. That's there for you to copy the content and use it in the PR, after that can be deleted 🧹

File: code/switch.js
Comment (AntonellaMorittu) [inline code/switch.js:1]: Please format your code to have the right spaces and indentation 👍

**Review by AntonellaMorittu — APPROVED** [review]
Hey Wen, good job this week 🚀 the code meets the requirements and stretch goals, that's great!  The app has a good flow and gives clear instructions to the user, bravo 👏  I left some comments specifically on some parts, please take a look at them :) Thank you ⭐

**Comment by wwenzz** [comment]
Hey Antonella, i removed the files not in use and aligned the space and indentation.

**Pull request closed without merging**